plugins/docker/docker_cpu

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/perl -w
```

Comment:
Instead of using commandline flags to enable warnings, please do:

    use strict;
    use warnings;